### ISSUE TITLE
Send raw file

### ISSUE THREAD

**Issue by kspeakman** [Zaid-Ajaj/Fable.SimpleHttp#16]
Is there a way in SimpleHttp to send a raw file instead of wrapping it in form data? The equivalent JavaScript looks like this:

```javascript
var input = ...;
var file = input.files[0];
xhr.send(file);
```

My exploration through the SimpleHttp API has found the way to do it through FormData and through the FileReader API. But these have unfavorable trade-offs in my situation. FormData requires me to do extra processing on the server side. FileReader has extra/unnecessary steps compared to sending the File object directly. Plus FileReader may have additional concerns with browser memory consumption.

I could submit a PR for this if interested. I figured I would just add another BodyContent type such as RawFile that accepts a File argument. Complication: we are currently using SimpleHttp 1.0.0 and Fable 2.0.3. The code base has been very active so I haven't yet braved the Fable 3 upgrade. I would understand if you were not interested in releasing a v1 patch. I can still submit a PR either way.

**Comment by Zaid-Ajaj** [comment]
Hello Kasey, indeed the current way of sending a file is using `formData` and as you have figured out, the simpler solution is to add another case for `BodyContent`. If you want to take a stab at it, please do :pray: that would be awesome and I would make sure to push these changes ASAP into a new package release.

However, the v1 patch is not doable for me as I can't maintain several versions of SimpleHttp and other libraries. You could either fall back to XHR to send the files until the Fable 3 upgrade but I recommend to make time for such upgrade to make it easier for your team moving forward and also easier to library maintainers :smile:

**Comment by kspeakman** [comment]
> However, the v1 patch is not doable for me

I understand completely. No worries.

**Comment by Zaid-Ajaj** [comment]
Fixed by #17

**Comment by kerams** [comment]
But `File` is an instance of `Blob` , so `BodyContent.Binary` works for both (I tried it) and you don't need another DU case.

**Comment by kspeakman** [comment]
🤣🤣🤣 I visited the MDN File page yesterday to implement this and did not notice the paragraph explaining that.

I notice that the new version is already out on NuGet, but currently zero downloads. I'm good with reverting this if feasible.

In case that turns out to be difficult, well at least sending a File will be more discoverable in SimpleHttp, even if technically unnecessary. 😀

**Comment by Zaid-Ajaj** [comment]
No worries @kspeakman I have reverted the change and published version 3.0.0, thanks a lot @kerams for the tip :heart: 

> I wasn't sure what the SemVer protocol should be here for reverting a PR, since it was a breaking change, I incremented the major version to 3.0 feedback welcome
